modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift-sdn/configuring-egress-firewall.adoc
+
+[id="nw-egressnetworkpolicy-about_{context}"]
+= How an egress firewall works in a project
+
+As an {product-title} cluster administrator, you can use an _egress firewall_ to
+limit the external IP addresses that some or all Pods can access from within the
+cluster. Egress firewall policy supports the following scenarios:
+
+- A Pod can only connect to internal hosts, and cannot initiate connections to
+the public Internet.
+- A Pod can only connect to the public Internet, and cannot initiate connections
+to internal hosts that are outside the {product-title} cluster.
+- A Pod cannot reach specified internal subnets or hosts outside the {product-title} cluster that should be unreachable.
```

Comment:
I suspect another common case will be:

- A pod can only connect to certain specified outside hosts, and cannot initiate connections to any other external hosts.